src/meta/server_state_restore.cpp

```diff
@@ -214,28 +214,25 @@ void server_state::on_query_restore_status(configuration_query_restore_rpc rpc)
     response.err = ERR_OK;
 
     std::shared_ptr<app_state> app = get_app(request.restore_app_id);
-    if (app == nullptr) {
-        response.err = ERR_APP_NOT_EXIST;
+    CHECK(nullptr != app, "app must be valid");
```

Comment:
shared_ptr重载了bool operator(),可以直接
```suggestion
    CHECK(app, "app must be valid");
```

Reply:
done